src/main/java/com/google/api/codegen/config/GrpcGapicRetryMapping.java

```diff
@@ -151,11 +155,17 @@ private static void applyRetryPolicyToName(
 
   private static RetryParamsDefinitionProto.Builder retryPolicyToParamsBuilder(
       RetryPolicy retryPolicy, long timeout, String policyName) {
+    // If the timeout is 0, make the timeout multiplier 0 as well, otherwise
```

Comment:
Is it necessary to set them both to 0? My naive understanding of this suggests that if timeout is 0 it does not matter what the multiplier is, because it will be multiplied by 0, so the total result will always be 0 anyways (well, unless the multiplier is infinity, but it is not practically possible).

Reply:
It was more of an aesthetic thing - if the timeout was 0, make the multiplier 0 too. I will revert this and have it just default to 1 regardless.

Reply:
Done.